spec/factories/test.html

```diff
@@ -0,0 +1,1917 @@
+<!DOCTYPE html>
```

Comment:
this ain't a good place for this file
`spec/factories` is a place only for `FactoryBot` factories

this file is more like a fixture, so put it within `spec/fixtures`